core/src/main/java/tc/oc/pgm/PGMConfig.java

```diff
@@ -196,6 +197,7 @@ public final class PGMConfig implements Config {
     this.maxExtraVotes = parseInteger(config.getString("votes.max-extra-votes", "5"));
 
     this.minPlayers = parseInteger(config.getString("join.min-players", "1"));
+    this.endEmptyMatches = config.getBoolean("join.end-empty-matches", true);
```

Comment:
```suggestion
    this.endEmptyMatches = config.getBoolean("join.end-empty-matches", minPlayers == 0);
```
This would keep the default in-line with current behavior (where min players=0 is used to avoid it ending when empty)